common/tools/dev-tool/src/util/browserRelayServer.ts

```diff
@@ -35,6 +35,7 @@ function buildServer(app: Express) {
   app.use((_req, res, next) => {
     res.set("Access-Control-Allow-Methods", "GET, PUT");
```

Comment:
Do we know if we need to allow other methods like POST, PATCH, etc? 

Not blocking, we can tackle that when if/when we need to

Reply:
I don't think we need to. This server only supports "Get" to get a token & check the server status and "Put" to create a new credential iiuc.